tests/spread/integration/fontconfig/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Integration tests for fontconfig
+
+execute: |
+  rootfs="$(install-slices bash_bins fontconfig_bins)"
+
+  mkdir -p $(rootfs)/tmp
+
+  [[ $(chroot "$rootfs" /usr/bin/bash -c 'HOME=/tmp fc-cache && echo $?') -eq 0 ]]
```

Comment:
Could you please test every binary being delivered by that slice? This is really to make sure the slice brings the necessary dependencies for each of them

Reply:
I checked with ldd (but I will have to slice it for spread, so will try to do that without it :)